tests/src/test/scala/cats/tests/SyntaxSuite.scala

```diff
@@ -299,9 +299,17 @@ object SyntaxSuite extends AllSyntaxBinCompat with AllInstances with AllSyntax {
 
     val fab = mock[F[A, B]]
     val gfcd = fab.bitraverse(f, g)
+    val gfad = fab.traverse(g)
+    val gfcb = fab.leftTraverse(f)
 
     val fgagb = mock[F[G[A], G[B]]]
     val gfab = fgagb.bisequence
+
+    val fgab = mock[F[G[A], B]]
+    val gfab = fgab.leftSequence
+
+    val fagb = mock[F[A, G[B]]]
+    val gfab = fagb.sequence
```

Comment:
`gfab` is defined twice

Reply:
good catch, I was concentrating too much on getting the names right to notice I'd repeated some